src/utilities/routerNavigation.spec.ts

```diff
@@ -2,21 +2,14 @@ import { expect, test, vi } from 'vitest'
 import { createRouterNavigation } from '@/utilities/routerNavigation'
 import { random } from '@/utilities/testHelpers'
 
-test('Browser like navigation is not supported', () => {
-  const onAfterLocationUpdate = vi.fn()
-  const navigation = createRouterNavigation({ onAfterLocationUpdate })
-
-  expect(() => navigation.back()).toThrowError()
-  expect(() => navigation.forward()).toThrowError()
-  expect(() => navigation.go(1)).toThrowError()
-})
```

Comment:
But now it is!

Reply:
are there even more .browser test suites that no longer NEED to be?

Reply:
I went through the `.browser.spec.ts` files and cleaned up what could.